code/game/cas_manager/datums/cas_fire_envelope.dm

```diff
@@ -259,7 +259,26 @@
 		mission_error = "Target is off bounds or obstructed."
 		return
 	change_current_loc(target_turf)
-	playsound(target_turf, soundeffect, 70, TRUE, 50)
+	playsound(source = target_turf, soundin = soundeffect, vol = 70, vary = TRUE, sound_range = 50, falloff = 8)
+
+	var/relative_dir
+	for(var/mob/M in range(15, target_turf))
+		if(get_turf(M) == target_turf)
+			relative_dir = 0
+		else
+			relative_dir = Get_Compass_Dir(M, target_turf)
+
+		var/ds_identifier = "LARGE BIRD"
+		var/fm_indentifier = "EXECUTE FIREMISSION"
+		if (M.mob_flags & KNOWS_TECHNOLOGY)
+			ds_identifier = "DROPSHIP"
+			fm_indentifier = "SPIT FIRE"
```

Comment:
```suggestion
			fm_indentifier = "EXECUTE FIREMISSION"
```

If this is for the technologically savvy shouldn't this one be `EXECUTE FIREMISSION`?

Reply:
jeh I fucking swaped it for those who know tech and who do not xD